src/main/java/com/github/cfogrady/dim/modifier/controls/ImageIntListView.java

```diff
@@ -0,0 +1,95 @@
+package com.github.cfogrady.dim.modifier.controls;
+
+import javafx.collections.ObservableList;
+import javafx.geometry.Insets;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.scene.text.Text;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import javafx.geometry.Pos;
+import javafx.geometry.VPos;
+import javafx.geometry.HPos;
+
+@Slf4j
+public class ImageIntListView extends ListView<ImageIntListView.ImageIntPair> {
+
+    public ImageIntListView() {
+        super();
+    }
+
+    public ImageIntListView(ObservableList<ImageIntPair> valueLabels, double imageScaler, Background background, String noneText) {
+        super();
+        this.setItems(valueLabels);
+        this.setCellFactory(lv -> new ImageIntCell(imageScaler, noneText, background));
+    }
+
+    public void initialize(ObservableList<ImageIntPair> valueLabels, double imageScaler, Background background, String noneText) {
+        this.setItems(valueLabels);
+        this.setCellFactory(lv -> new ImageIntCell(imageScaler, noneText, background));
+    }
+
+    @Data
+    @EqualsAndHashCode(callSuper = true)
+    public static class ImageIntPair extends LabelValuePair<Integer, Image> {

```

Comment:
Since this is the exact same as the ImageIntPair in ImageIntComboBox, I'd prefer if we pull this common class out to be used by both instead of duplicating it. If that's done, it will also mean the existing createImageValuePairs method of SpriteImageTranslator will work for both so we can also remove that duplicated code.